test/contract/interactions/api/cdo.js

```diff
@@ -0,0 +1,75 @@
+const {
```

Comment:
I like the way the interactions are split out into separate files, now that there are more interactions

Reply:
Thanks